cli/cmd/config.go

```diff
@@ -119,7 +120,8 @@ func setConfigProperty(config *common.OdigosConfiguration, property string, valu
 
 	case consts.TelemetryEnabledProperty, consts.OpenshiftEnabledProperty, consts.PspProperty,
 		consts.SkipWebhookIssuerCreationProperty, consts.AllowConcurrentAgentsProperty,
-		consts.KarpenterEnabledProperty, consts.RollbackDisabledProperty:
+		consts.KarpenterEnabledProperty, consts.RollbackDisabledProperty,
+		consts.RolloutConfigurationAutomaticRolloutDisabledProperty:
```

Comment:
nit: the name too long and Rollout duplicates
```suggestion
		consts.AutomaticRolloutDisabledProperty:
```